fix(pornpics): properly configure language settings

marioplus opened the pull request:
- Implement language configuration via SourceFactory

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

marioplus committed:
fix(pornpics): properly configure language settings

- Implement language configuration via SourceFactory

AwkwardPeak7 reviewed (commented):
whats the point of factory if the content will be same across all three? generally factory source is used to filter based on language when it comes to entries during browsing or chapters

AwkwardPeak7 commented on src/all/pornpics/src/eu/kanade/tachiyomi/extension/all/pornpics/PornPicsFactory.kt:
```diff
@@ -0,0 +1,12 @@
+class PornPicsFactory : SourceFactory {
+
+    override fun createSources() = listOf(
+        PornPics("all"),
```
if this is only for intl language, then no need for `all`? since translations are for `en` and `zh`. 
you can assign existing `all` id to `en`

```kt
override val id = when (lang) {
    "en" -> <existing id>
    else -> super.id
}
```
get existing id from https://github.com/keiyoushi/extensions/blob/repo/index.json

marioplus replied:
Do I need to bind ID and Lang?

AwkwardPeak7 replied:
so in factory, only return en and zh, and in the source class, set en to old all id

marioplus committed:
fix(pornpics): properly bind ID and language settings

- Associate language codes with correct source IDs
- Use 'all' ID for English (en) language

AwkwardPeak7 commented on src/all/pornpics/src/eu/kanade/tachiyomi/extension/all/pornpics/PornPicsFactory.kt:
```diff
@@ -0,0 +1,12 @@
+class PornPicsFactory : SourceFactory {
+
+    override fun createSources() = listOf(
+        PornPics("all"),
```
```suggestion
```

marioplus committed:
fix: remove all lang
